### PR TITLE
Add cardpointers extension

### PR DESCRIPTION
**Pull request by emcro**
## Description

Search for any credit card or Amex/Chase offer detail, right in Raycast.

Copy and paste any detail, open the card or offer right int the CardPointers app, or on the web.

Powered by CardPointers.

## Screencast

https://github.com/raycast/extensions/assets/689374/21f592c3-bd4b-4efb-85c6-368edeb07258

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/cardpointers/src/top-cards.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                <ActionPanel.Section />
+
+                <Action.CopyToClipboard
+                  title="Copy Link"
+                  shortcut={{ modifiers: ["cmd"], key: "l" }}
+                  content={`https://cardpointers.com/cards/${card.slug}/`}
+                />
+                <Action.CopyToClipboard
+                  title="Copy Card Title"
+                  shortcut={{ modifiers: ["cmd"], key: "t" }}
```
Comment (pernielsentikaer) [inline extensions/cardpointers/src/top-cards.tsx:76]: Maybe also add `shift` as modifier, `⌘+t` is a default Raycast shortcut

File: extensions/cardpointers/src/top-cards.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                  title="Open in Browser"
+                  url={`https://cardpointers.com/cards/${card.slug}/?b=1`}
+                  shortcut={{ modifiers: ["cmd"], key: "b" }}
+                />
+
+                <ActionPanel.Section />
+
+                <Action.CopyToClipboard
+                  title="Copy Link"
+                  shortcut={{ modifiers: ["cmd"], key: "l" }}
```
Comment (pernielsentikaer) [inline extensions/cardpointers/src/top-cards.tsx:71]: Maybe also add `shift` as modifier

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/cardpointers/src/top-cards.tsx (hunk 1 above)
Comment (emcro) [reply extensions/cardpointers/src/top-cards.tsx:76]: Hi, I'm not seeing Cmd-T do anything in Raycast, and it works as-is. Is this a must-change for some future reason, or can I leave as-is? It's cleaner to have them all be simple Cmd-shortcuts. Thanks for reviewing!

**Comment by emcro** [comment]
Hi, I'm a bit confused by these changes TBH.

1. The first screenshot was removed, but shows the full list of commands which is helpful to users to see what the extension does; this matches a similar screenshot featured in most extensions, such as https://www.raycast.com/screen-studio/screen-studio. If this needs to be removed anyhow, do I need to rename each image file so that it's 1-5 then, or is 2-6 okay for naming with your build process for the store?

2. The `navigationTitle` was removed with https://github.com/raycast/extensions/pull/6778/commits/75c6da1a3d985f95d3837bde11c4b088c74ddf4a but it's stated in the docs to use the navigationTitle when it's in a detail screen, which is what that view is, and gives context for what the user is looking at.

3. There are now two `subtitle` entries for the `top-cards` shortcut in `package.json`; both of those two subtitles give more context about those two commands and include the extension name in there, so I would prefer to retain those and revert your commit if that's acceptable.
 
5. Please see inline comments about the modifiers; Cmd-L and Cmd-T don't seem to be reserved for Raycast, they don't do anything anywhere in the app for me, and both shortcuts work as-is. Do I *need* to add the Shift modifier to all of those, or is that only a suggestion? I prefer the consistency with the rest of the extension's shortcuts and it's one less keypress for users.


Thank you,

Emmanuel

**Comment by pernielsentikaer** [comment]
Hi 👋 

I can see that I was a bit too fast with some of them; sorry for the inconvenience.

1) You have the possibility to show some of the core features of the extension, whereas the one you mentions is all `no-view` and doesn't have that possibility 🙂 The naming is acceptable as is 🔥 

2) Correct, I was way too fast here, I added it back and enhanced it a bit

3) It makes sense as it is right now, are you okay with this one?

> Use subtitles to add context to your command. Usually, it's an app or service name that you integrate with. It makes command names more lightweight and removes the need to specify a service name in the command title.

4) Let's keep them as is 🙂

**Comment by emcro** [comment]
Thank you for the extra details. I implemented all of the changes (and applied them to the Offer Details component for consistency as well) and re-ordered the screenshots to tell the right story (removing the one with the list of commands as requested). Please let me know if you need any other changes.

**Pull request merged**